EFCore-Adapter/CasbinDbAdapter.cs

```diff
@@ -133,24 +144,81 @@ public virtual void AddPolicy(string section, string policyType, IList<string> r
 
         public virtual async Task AddPolicyAsync(string section, string policyType, IList<string> rule)
         {
+            if (rule is null || rule.Count is 0)
+            {
+                return;
+            }
+
             var casbinRule = CasbinRuleExtenstion.Parse<TCasbinRule>(policyType, rule);
             casbinRule = OnAddPolicy(section, policyType, rule, casbinRule);
             await CasbinRules.AddAsync(casbinRule);
             await DbContext.SaveChangesAsync();
         }
+        
+        public void AddPolicies(string section, string policyType, IEnumerable<IList<string>> rules)
+        {
+            if (rules is null)
+            {
+                return;
+            }
+
+            var rulesArray = rules as IList<string>[] ?? rules.ToArray();
+            if (rulesArray.Length is 0)
+            {
+                return;
+            }
+
+            var casbinRules = rulesArray.Select(r => 
```

Comment:
Should we avoid adding duplicate rules into the database? If a rule already exists, should we add it twice? Duplicate rules may not affect the enforce result, so you may ignore this comment if you think it's ok.

Reply:
I think we should avoid it and I will recommit later.

Reply:
I think the duplicate rules should be removed on InternalEnforcer at Casbin.NET, we need not avoid this problem at Adapter.

Reply:
In my opinion, the duplicate rules should be removed on InternalEnforcer at Casbin.NET, we need not avoid this problem at Adapter.